src/Evolution/Systems/Cce/WorldtubeDataManager.cpp

```diff
@@ -181,7 +181,7 @@ bool MetricWorldtubeDataManager::populate_hypersurface_boundary_data(
   // coefficients. This is what the boundary data calculation utility takes
   // as an input, so we now hand off the control flow to the boundary and
   // gauge transform utility
-  if (buffer_updater_->radial_derivatives_need_renormalization()) {
+  if (buffer_updater_->has_version_history()) {
```

Comment:
Sanity check question: there are two diffs above where a `false` radial_derivatives_need_renormalization is replaced by a `true` has_version_history. That makes it seem like the new variable is the logical inverse of the old variable. Should this use change?

Reply:
I agree -- some of this logic is erroneously reversed. I'll fix that.